core/classes/Misc/ErrorHandler.php

```diff
@@ -110,14 +110,14 @@ public static function catchException(?Throwable $exception, ?string $error_stri
             $language = new Language('core', 'en_UK');
         }
 
-        $detailed_error = defined('DEBUGGING');
-        $can_generate_debug = defined('DEBUGGING');
+        $detailed_error = defined('DEBUGGING') && DEBUGGING;
+        $can_generate_debug = defined('DEBUGGING') && DEBUGGING;
 
         try {
             $user = new User();
             $detailed_error |= $user->isLoggedIn() && $user->hasPermission('admincp.errors');
             $can_generate_debug |= $user->hasPermission('admincp.core.debugging');
-        } catch (Error $ignored) {
+        } catch (Error | RuntimeException $ignored) {
```

Comment:
Weird, thought this was here already. What's `RuntimeException` for exactly - totally forget

Reply:
This should be `Exception`, all exceptions inherit from `Exception` and all errors inherit from `Error`. We probably shouldn't be catching errors.